test/e2e/logfilesmetricexporter/metrics_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/bash
```

Comment:
Can we do this as a functional test to make testing easier?

Reply:
There is no support for `logfilesmetricexporter` in the functional framework. We will need to determine the viability of adding/creating it for our test.

Reply:
Given the work required, I will defer to leaving this as an e2e test